pyro/infer/inspect.py

```diff
@@ -339,6 +366,16 @@ def generate_graph_specification(model_relations: dict) -> dict:
         rv for rv in model_relations["sample_sample"] if rv not in plate_rvs
     ]  # RVs which are in no plate
 
+    params = set()
+    if render_params:
+        for rv, params_list in model_relations["sample_param"].items():
+            for param in params_list:
+                params.add(param)
+        params = list(params)
```

Comment:
I think you can simply delete this line, right?

Reply:
Yes, I removed that line.